sdks/go/pkg/beam/runners/universal/runnerlib/compile.go

```diff
@@ -55,10 +55,10 @@ func BuildTempWorkerBinary(ctx context.Context) (string, error) {
 // BuildWorkerBinary creates a local worker binary for linux/amd64. It finds the filename
 // by examining the call stack. We want the user entry (*), for example:
 //
-//   /Users/herohde/go/src/github.com/apache/beam/sdks/go/pkg/beam/runners/beamexec/main.go (skip: 2)
-// * /Users/herohde/go/src/github.com/apache/beam/sdks/go/examples/wordcount/wordcount.go (skip: 3)
-//   /usr/local/go/src/runtime/proc.go (skip: 4)      // not always present
-//   /usr/local/go/src/runtime/asm_amd64.s (skip: 4 or 5)
+//	  /Users/herohde/go/src/github.com/apache/beam/sdks/go/pkg/beam/runners/beamexec/main.go (skip: 2)
+//	* /Users/herohde/go/src/github.com/apache/beam/sdks/go/examples/wordcount/wordcount.go (skip: 3)
+//	  /usr/local/go/src/runtime/proc.go (skip: 4)      // not always present
+//	  /usr/local/go/src/runtime/asm_amd64.s (skip: 4 or 5)
```

Comment:
This was very broken before - https://pkg.go.dev/github.com/apache/beam/sdks/v2@v2.43.0/go/pkg/beam/runners/universal/runnerlib#BuildWorkerBinary

Now it renders as:

<img width="1177" alt="image" src="https://user-images.githubusercontent.com/42773683/205669881-e36fa8ad-d908-4ba4-b9c8-63f7b3dd7ec1.png">